api/v1alpha/README.md

```diff
@@ -472,11 +472,8 @@ Class | Method | HTTP request | Description
  - [CloudProviderAccess](docs/CloudProviderAccess.md)
  - [CloudProviderAccessAWSIAMRole](docs/CloudProviderAccessAWSIAMRole.md)
  - [CloudProviderAccessDataLakeFeatureUsage](docs/CloudProviderAccessDataLakeFeatureUsage.md)
- - [CloudProviderAccessDataLakeFeatureUsageAllOf](docs/CloudProviderAccessDataLakeFeatureUsageAllOf.md)
```

Comment:
CC @AndreiMatei2206 I believe that this was something you were looking for fixing. 
After adding oneOf to every discriminator - the discriminator link is no longer driving relationships. That means a much simpler result model and we do not have duplicate templates. 
This also resolves problems with os.Time